apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -10718,3 +10771,6 @@ body.prevent-body-scroll {
     display: none!important;
 }
 }
+.daterangepicker{
```

Comment:
Do use dash-case

Reply:
after global search found 871 results into 116 file including TS and other BS files, It seems risky to correct this class. Need to ignore.

Reply:
Okay